tests/test_recipient_csv.py

```diff
@@ -1302,3 +1302,24 @@ def test_email_validation_speed():
     )
     for row in recipients:
         assert not row.has_error
+
+
+@pytest.mark.parametrize('should_validate', [True, False])
```

Comment:
Very optional, you could also add an explicit test case in here for the default, ie when `should_validate` has not been set. Similar to https://github.com/alphagov/notifications-utils/pull/946/files#diff-936d1cd89027c14eeb7a43e17df32fa0234db7b2313af63b9b1a8e69e689bdf3R252

However, I don't mind if we don't as I think the rest of the tests in this file give us enough confidence that validation is happening when we don't include the `should_validate` argument